core/filenumber_checker.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+from pathlib import Path
+
+import pandas as pd
+from moviepy.video.io.VideoFileClip import VideoFileClip
+
+from utils import check_if_same, convert_to_path
+
+class Framenumberchecker():
```

Comment:
Could be implemented into the pipeline after synchronisation to make sure that
- synchro worked
- no frame drops in videos

Reply:
Even before after collecting metadata to ensure that the data quality is good enough

Reply:
Before synchronising, videos with different fps will have different numbers of frames. You could also go for duration? (number of frames/fps)?